src/DurableTask.SqlServer/SqlOrchestrationServiceSettings.cs

```diff
@@ -102,6 +125,14 @@ public SqlOrchestrationServiceSettings(string connectionString, string? taskHubN
         [JsonProperty("createDatabaseIfNotExists")]
         public bool CreateDatabaseIfNotExists { get; set; }
 
+        /// <summary>
+        /// Gets or sets a flag indicating wether task hub is created for multi-tennant naming
+        /// (derived from sql server user name) or for application specified naming.
+        /// </summary>
+        /// <seealso cref="SqlOrchestrationService.CreateAsync(bool)"/>
+        [JsonProperty("createMultiTennantTaskHub ")]
+        public bool CreateMultiTennantTaskHub { get; set; } = true;
```

Comment:
According to Wikipedia, "[multitenant](https://en.wikipedia.org/wiki/Multitenancy)" is a single word with just one "n". This is also the spelling we use in our documentation. Let's update all the new code in this PR to use that spelling.

Example here:
```suggestion
        public bool CreateMultitenantTaskHub { get; set; } = true;
```

Reply:
Fixed